api/constants.ts

```diff
@@ -16,7 +16,7 @@ export const APIEndpoints = {
     BASE_ENDPOINT: "/links",
     CREATE_LINK: "/",
     UPDATE_LINK: "/",
-    GET_LINK: "/:shortCode([A-Za-z]{5})",
+    GET_LINK: "/:shortCode([A-Za-z]{4,120})",
```

Comment:
`/:shortCode([A-Za-z]{4,120})` supports only alphabets. Short codes with `-`, `_` and numbers should also be allowed.

Reply:
oop, fixed 👍